app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -726,6 +726,14 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
 
     @VisibleForTesting
     internal fun reportHomeScreenMetrics(settings: Settings) {
+        CustomizeHome.openingScreen.set(
+            when {
+                settings.alwaysOpenTheHomepageWhenOpeningTheApp -> "homepage"
+                settings.alwaysOpenTheLastTabWhenOpeningTheApp -> "last tab"
+                settings.openHomepageAfterFourHoursOfInactivity -> "homepage after four hours"
+                else -> ""
+            }
+        )
         components.analytics.experiments.register(object : NimbusInterface.Observer {
```

Comment:
I was unsure what's going on here with the Nimbus code, so I chose to put this particular metric outside of it.

Reply:
This observer waits for Nimbus (our experimentation framework) to finish it's async startup before we send back telemetry results which may be based an experiment group the user is in.

The questions worth asking that case is, are these telemetry probes you're collection dependant on an experiment, are the options visible/invisible, etc.

Reply:
If the answer is, they don't depend on an experiment. They should be fine there, although I'd leave that as a comment in the code because it would definitely look odd to the next person coming by and wondering the same thing.

Reply:
Yeah, that's where my uncertainly is mainly coming from. I'm primarily unsure whether the experiment code here shows/hides stuff (and maybe is even still running), in which case I'd need to put my new probe with the existing ones.

Reply:
> (and maybe is the experiment even still running)

It's harder to determine future state but we can check the current state - so let's do that!

I've elaborated my investigation below if that helps you in the future on how to find the answers you're looking for. 🙂 

Let's look at all the references to `alwaysOpenTheHomepageWhenOpeningTheApp`, `alwaysOpenTheLastTabWhenOpeningTheApp`, `openHomepageAfterFourHoursOfInactivity`: they all seem to come from `Settings.shouldStartOnHome` and that is only used on the [`HomeActivity`](https://github.com/mozilla-mobile/fenix/blob/c026d8b28addbf0eab914f8d777339d38061e347/app/src/main/java/org/mozilla/fenix/HomeActivity.kt#L990-L999).

In that block itself we see a feature flag, `FeatureFlags.showStartOnHomeSettings` which is defaulted to on. Looking at other references to the HomeActivity method there, I see none that suspect they are related to feature flags or nimbus.

So let's look at the `FeatureFlags.showStartOnHomeSettings`: with a [git blame on the file](https://github.com/mozilla-mobile/fenix/blame/main/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt#L34), we can see that we flipped the flag true as part of our 94 release. With [94 now in release](https://whattrainisitnow.com/), we typically remove the feature flag since we rarely have a need to keep it around for much longer. 

This tells me that we should be fine with collecting the preference where you currently have, of even after Nimbus is done loading, because the value isn't dependant on it.

Reply:
This makes sense. Thank you so much for taking the time to explain your thought process of your exploration and for the additional context about the retirement of feature flags.